Snakefile

```diff
@@ -135,8 +137,8 @@ rule download_cgi:
     output: f'{DATA_DIR}/cgi/cgi_biomarkers_per_variant.tsv'
     shell: dedent(f'''\
         cd {DATA_DIR}/cgi
-        wget https://www.cancergenomeinterpreter.org/data/cgi_biomarkers_latest.zip
-        unzip cgi_biomarkers_latest.zip
+        wget https://www.cancergenomeinterpreter.org/data/cgi_biomarkers_20180117.zip
+        unzip cgi_biomarkers_20180117.zip
```

Comment:
why is it being fixed at this 2018 version?

Reply:
They removed the "latest" version and 2018 is the latest version they have. This source is no longer being updated as far as I know